libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -435,6 +435,18 @@ class Strings {
         assertPrints(matchDetails(inputString, toFind, 10), "Searching for 'ever' in 'Never ever give up' starting at position 10: Not found")
     }
 
+    @Sample
+    fun find() {
```

Comment:
This sample is not compilable, please run it locally before submitting the updated version.